cypress/support/pages/signIn.pageObject.js

```diff
@@ -3,16 +3,19 @@ import PageObject from '../PageObject';
 class SignInPageObject extends PageObject {
   url = '/user/login';
 
-  get emailField() {
-    return cy.getByDataCy('email-sign-in');
+  emailField(email) {
+    cy.getByDataCy('email-sign-in')
+      .type(email);
   }
 
-  get passwordField() {
-    return cy.getByDataCy('password-sign-in');
+  passwordField(password) {
```

Comment:
Function names should typically be verbs or verb phrases. Consider renaming 'passwordField' to 'enterPassword' or something similar that reflects the function's action.